fab/fabfile.py

```diff
@@ -422,6 +428,7 @@ def _deploy_without_asking():
         staticfiles.prime_version_static,
         db.ensure_preindex_completion,
         # handle static files
+        db.ensure_checkpoints_safe,
```

Comment:
nit: should put this above the comment though i wouldn't be against removing the comment completely